lib/rentlinx/modules/amenity_client_methods.rb

```diff
@@ -47,7 +58,11 @@ def unpost_amenity(amenity)
     # post_amenities should only be called from property or unit, so we don't need
     # to handle multiple properties
     def post_property_amenities(amenities)
-      request('PUT', "properties/#{amenities.first.propertyID}/amenities", amenities.map(&:to_hash))
+      post_amenities_for_property_id(amenities.first.propertyID, amenities)
+    end
+
+    def post_amenities_for_property_id(id, amenities)
```

Comment:
Nevermind, it's not.  I assumed post_property_amenities was public and that this method was as well, but they're both private.